### DISCUSSION TITLE
Show more than 15 icons on profile page

### DISCUSSION THREAD

**Discussion by jamesdlow** [elrumo/macOS_Big_Sur_icons_replacements#1336] [Ideas]
Hi, I noticed the profile only shows the last 15 icons uploaded. Could this be expanded to show all the icons uploaded by each author? It could have an infinite scroll if there are too many for the initial load.

**Comment by jamesdlow** [comment]
I just tried on mobile, you do have a infinite scroll, the issue is it's not triggered if the initial set of loaded images does not extend to the bottom of the screen:

![Screen Shot 2021-12-03 at 12 41 56 PM png](https://user-images.githubusercontent.com/114436/144520324-d4d63fe2-5662-4e70-aab8-22b684be46dd.jpg)

**Reply by elrumo** [reply]
Thanks for pointing out the bug, that is indeed the issue. At the moment, the infine scroll is triggered when the user scrolls, so if there is nothing to scroll, it dones't get triggered. I'll try to fix it.

**Reply by elrumo** [reply]
Should be fixed on [beta.macosicons.com](beta.macosicons.com), I'll push it to the main branch later this month:
https://github.com/elrumo/macOS_Big_Sur_icons_replacements/commit/4a0c124744a34ef65fbc29b2b8139e177528b563
